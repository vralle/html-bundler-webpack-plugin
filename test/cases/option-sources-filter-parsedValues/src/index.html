<!DOCTYPE html>
<html>
  <head>
    <title>Test</title>
  </head>
  <body>
    <!-- this data-srcset should be resolved  -->
    <img src="@images/apple.png" data-srcset="@images/fig.png, @images/fig2.png 100w, @images/fig3.png 1.5x">
    <!-- this data-srcset should not be resolved  -->
    <img src="@images/pear.png?size=300" data-srcset="@images/lemon.png?size=200, @images/lemon2.png 100w, @images/lemon3.png 1.5x">
  </body>
</html>