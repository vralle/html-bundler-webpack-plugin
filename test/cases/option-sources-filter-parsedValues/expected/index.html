<!DOCTYPE html>
<html>
  <head>
    <title>Test</title>
  </head>
  <body>
    <!-- this data-srcset should be resolved  -->
    <img src="assets/img/apple.02a7c382.png" data-srcset="assets/img/fig.c6809878.png, assets/img/fig2.c6809878.png 100w, assets/img/fig3.c6809878.png 1.5x">
    <!-- this data-srcset should not be resolved  -->
    <img src="@images/pear.png?size=300" data-srcset="@images/lemon.png?size=200, @images/lemon2.png 100w, @images/lemon3.png 1.5x">
  </body>
</html>